### PR TITLE
Add remaining prepare endpoints and refactor

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

File: api/lib/constants.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    name: 'disallow_xrp',
+    value: ripple.Remote.flags.account_root.DisallowXRP
+  },
+  DisableMaster: {
+    name: 'disable_master',
+    value: ripple.Remote.flags.account_root.DisableMaster
+  },
+  NoFreeze: {
+    name: 'no_freeze',
+    value: 0x00200000
```
Comment (geertweening) [inline api/lib/constants.js]: ripple.Remote.flags.account_root.NoFreeze

`js/ripple/remote.js:223:    NoFreeze: 0x00200000,`

File: api/lib/constants.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  },
+  GlobalFreeze: {
+    name: 'global_freeze',
+    value: 0x00400000
```
Comment (geertweening) [inline api/lib/constants.js]: ripple.Remote.flags.account_root.GlobalFreeze

**Comment by lumberj** [comment]
LGTM

**Comment by geertweening** [comment]
LGTM

**Pull request merged**